package/yast2-sudo.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Dec 31 10:07:40 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Do not truncate the sudoers file after write changes
+  (bsc#1156929).
+- 4.0.1
```

Comment:
Seems that [`4.0.1` already exists](https://github.com/yast/yast-sudo/blob/4bbcc3f5d1cc4d04097b980af3dbd1b4f813a725/package/yast2-sudo.changes#L26) :sob: